### PR TITLE
Fix internationalisation for Card component

### PR DESCRIPTION
**Pull request by victorhmp**
#### What is the purpose of this pull request?
Fix internationalisation for Card component.

#### What problem is this solving?
Card component was not internationalized.

#### How should this be manually tested?

https://victorhugo--storecomponents.myvtex.com/dev/io
(Change the language)

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Navbar.tsx [hunk 1]
```diff
@@ -58,4 +77,15 @@ const Navbar: FunctionComponent<Props & InjectedIntlProps> = ({
   )
 }
 
+function splitLocale(locale: string) {
```
Comment (matheusps) [inline react/Navbar.tsx]: Could just be:
```suggestion
const splitLocale = (locale: string) =>  locale.split('-')[0]
```
The code has the correct behavior, though. Changing or not is completely up to you 🙂

File: react/Navbar.tsx [hunk 2]
```diff
@@ -58,4 +77,15 @@ const Navbar: FunctionComponent<Props & InjectedIntlProps> = ({
...
+  return locale.split('-')[0]
+}
+
+function findLocale(locale: any) {
```
Comment (matheusps) [inline react/Navbar.tsx]: Same logic, could be shortened ⬆️

File: react/Navbar.tsx (hunk 1 above)
Comment (victorhmp) [reply react/Navbar.tsx]: Yep, good catch! 

I was using arrow functions right when I wrote this hahahah
I changed it to `function` just for better readability, as `function` allows me to put these declarations wherever I want in the file, which allowed me to declare these functions _after_ declaring the actual `FunctionComponent`, not before, as I would have to do if these were arrow functions.

Comment (matheusps) [reply react/Navbar.tsx]: Oh, okay

**Pull request merged**